defaults/main.yml

```diff
@@ -26,9 +25,9 @@ redis_dir: /var/lib/redis
 redis_maxclients: 10000
 
 # Max memory values
-redis_maxmemory: 128mb
-redis_maxmemory_policy: noeviction
-redis_maxmemory_samples: 3
+redis_maxmemory: 2gb
+redis_maxmemory_policy: volatile-ttl
+redis_maxmemory_samples: 5
```

Comment:
I'm a bit cautious about increasing redis max memory this much. 
However, switching to `volatile-ttl` max memory policy is a welcome change, especially in the context of raising `redis_maxmemory`. Having that said, Redis 4.0 offers the new `volatile-lfu` max memory policy, which, in theory, should be even more efficient for cache eviction.